Lib/test/test_mmap.py

```diff
@@ -839,7 +840,8 @@ def test_flush_return_value(self):
         mm.write(b'python')
         result = mm.flush()
         self.assertIsNone(result)
-        if sys.platform.startswith(('linux', 'android')):
+        if (sys.platform.startswith(('linux', 'android'))
+            and not in_systemd_nspawn_sync_suppressed()):
```

Comment:
Perhaps we could follow black here:

```suggestion
        if (
            sys.platform.startswith(("linux", "android"))
            and not in_systemd_nspawn_sync_suppressed()
        ):
```